modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/NodeLeftException.java

```diff
@@ -29,7 +29,7 @@ public class NodeLeftException extends IgniteException {
     private static final long serialVersionUID = 0L;
 
     /** Constructor. */
-    public NodeLeftException(String nodeName) {
-        super(NODE_LEFT_ERR, "Node left the cluster. Node: " + nodeName);
```

Comment:
What reason to move standart text message for the exception to invoke side? I prefer have message template inside a exceptions if it possible.

Reply:
it is not recommended to change the exception message in the constructor, because of this issue: https://issues.apache.org/jira/browse/IGNITE-20140

Reply:
Done